src/people.js

```diff
@@ -26,6 +26,19 @@
  * @returns {PeopleResponse}  - Processed information
 */
 
-function peopleInformation(people) { }
+function peopleInformation(people) { 
+  const listAge = people.map((person) => person.age);
+  const listHeight = people.map((person) => person.height);
+  const ageProm = listAge.reduce((a, b) => a + b) / people.length;
+  const heightProm = listHeight.reduce((a, b) => a + b) / people.length;
```

Comment:
Good move, but you iterated the array four times. 
Could you think of a way to iterate it only once?